src/main/java/com/denizenscript/depenizen/bukkit/bridges/MythicMobsBridge.java

```diff
@@ -206,7 +333,27 @@ public static void setMythicVariableMap(Entity entity, Map<String, Variable> map
         registry.putAll(map);
     }
 
+    public static ItemExecutor getItemManager() {
+        return MythicBukkit.inst().getItemManager();
+    }
+
+    public static Collection<String> getItemNames() {
```

Comment:
Feels a bit unnecessary to have this as a method, it's only used in once place and using `getItemManager().getItemNames()` directly isn't a lot longer/more complex or anything

Reply:
Still applies, this method feels redundant imo